spec/lib/ability_spec.rb

```diff
@@ -205,6 +205,14 @@
       it { is_expected.not_to be_allowed_to(:manage_billing, facility) }
       it { is_expected.not_to be_allowed_to(:transactions, facility) }
       it { is_expected.not_to be_allowed_to(:manage, Reservation) }
+
+      context "when show_estimates_option is enabled", feature_setting: { show_estimates_option: true } do
```

Comment:
Nit: This looks like something that can be defined in a shared example, same for the disabled context